e2e/render_test.go

```diff
@@ -42,10 +40,10 @@ func gather(t *testing.T, dir string) ([]string, []string, map[string]string) {
 	return settings, overrides, env
 }
 
-// checkRenderer returns false if appname requires a renderer that is not in enabled
+// checkRenderers returns false if appname requires a render that is not in enabled
```

Comment:
Are all the `renderer` -> `render` on purpose?
(or did you mean 🎅 🦌?)

Reply:
ah crap, search-and-replace small failure :)